cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -216,7 +216,7 @@ if(builtin_lzma)
       URL_HASH SHA256=a923ee68d836de5492d8de0fec467b9536f2543c8579ca11f4b5e6f46a8cda8c
       PREFIX LZMA
       INSTALL_DIR ${CMAKE_BINARY_DIR}
-      CONFIGURE_COMMAND ""
+      CONFIGURE_COMMAND "--disable-scripts"
```

Comment:
Please check carefully, you most probably don't want to add this here. `--disable-scripts` will be executed in the command prompt and it is obviously not a valid command...

Reply:
Indeed, it's in the wrong section. I'll fix this :)